manifests/kilo-k3s-flannel.yaml

```diff
@@ -83,14 +112,36 @@ spec:
         - name: kilo-dir
           mountPath: /var/lib/kilo
         - name: kubeconfig
-          mountPath: /etc/kubernetes/kubeconfig
+          mountPath: /etc/kubernetes
           readOnly: true
         - name: lib-modules
           mountPath: /lib/modules
           readOnly: true
         - name: xtables-lock
           mountPath: /run/xtables.lock
           readOnly: false
+      initContainers:
+      - name: generate-kubeconfig
+        image: busybox
```

Comment:
Could we re-use the Kilo image for these init-containers? We already do so for the `install-cni` init-containers, as the images include `/bin/sh`, so I'd like to keep it consistent.